src/css/main.css

```diff
@@ -71,6 +71,60 @@ button:hover{
     border:1px solid #f00;
 }
 
+.form-row {
+    display: flex;
+    width: "50%";
+}
+
+.form-row-extremes {
+    display: inline-block; 
+    width: 100%;
+}
+
+.form-group { 
+    flex:1;
+    padding-left:5px;
+    padding-right:5px;
+    padding-bottom: 0;
+}

```

Comment:
There a couple other modals that use form-group and these edits will also change how those look. See the "Help" modal for example. We should just make sure we know what this will all change.